test/plugins/node-workspace.ts

```diff
@@ -1115,4 +1115,131 @@ describe('NodeWorkspaceDependencyUpdates', () => {
       snapshot(snapPrefix + ' changes', stringifyActual(actualChanges));
     });
   });
+
+  it('can force local package linking with config', async function () {
+    const config: Config = {
+      'always-link-local': true,
+      packages: {}, // unused, required by interface
+      parsedPackages: [
+        {path: 'packages/pkgA', releaseType: 'node'},
+        {path: 'packages/pkgB', releaseType: 'node'},
+        {path: 'packages/pkgC', releaseType: 'node'},
+      ],
+    };
+    const github = new GitHub({
+      owner: 'fake',
+      repo: 'repo',
+      defaultBranch: 'main',
+    });
+    const mock = mockGithub(github);
+    // package C did not get a release-please updated but it depends on both
+    // A and B which did get release-please bumps so it should receive a
+    // patch bump
+    expectGetFiles(mock, [
+      ['packages/pkgA/package.json', false],
+      ['packages/pkgA/CHANGELOG.md', false],
+      ['packages/pkgB/package.json', false],
+      ['packages/pkgB/CHANGELOG.md', false],
+      [
+        'packages/pkgC/package.json',
+        prettyJsonStringify({
+          name: '@here/pkgC',
+          version: '3.3.3',
+          dependencies: {
+            '@here/pkgA': '^1.1.1',
+            '@here/pkgB': '^0.2.1',
+            anotherExternal: '^4.3.1',
+          },
+        }),
+      ],
+      [
+        'packages/pkgC/CHANGELOG.md',
+        '# Changelog' +
+          '\n\nAll notable changes to this project will be ' +
+          'documented in this file.' +
+          '\n\n### [3.3.3](https://www.github.com/fake/repo/compare' +
+          '/pkgC-v3.3.2...pkgC-v3.3.3) (1983-10-10)' +
+          '\n\n\n### Bug Fixes' +
+          '\n\n* We fixed a bug',
+      ],
+    ]);
+
+    // pkgA had a patch bump and pkgB had a minor bump from
+    // manifest.runReleasers()
+    const newManifestVersions = new Map([
+      ['packages/pkgA', '1.1.2'],
```

Comment:
maybe it's over-kill but just for my clarification: if you made this change:
```suggestion
      ['packages/pkgA', '2.0.0'],
```
then B's `@here/pkgA` would update to `^2.0.0` as well, correct? In otherwords, it's not just the "pre-major, breaking change version bump" that's handled but also the "post-major, breaking change version bump" (and I suppose the "pre-pre-major, major version bump" but I don't think we need to vet that here)?

Reply:
I believe you are correct here, but the node-workspace plugin makes an assumption that the version update for dependencies should always be `"patch"` here: https://github.com/googleapis/release-please/blob/master/src/plugins/node-workspace.ts#L165

Do you have any ideas how we'd determine the correct bump to make?

Reply:
That particular assumption is the following: unless otherwise bumped by actual conventional commits, a package should only bump by a `patch` since the only thing that's changing about it is dependency versions - even when a dependency version crosses a breaking change boundary. I think that still makes sense: just because a library you're using has some breaking change doesn't imply that the functionality you expose to users of your library has a breaking change. If it did, then you'd most likely have an actual conventional commit against your package with a `feat/fix!`

What I was driving at here was simply verifying that if conventional commits had pushed pkgA to `2.0.0`, then pkgB and pkgC would now point to to `'@here/pkgA': '^2.0.0'`. I'd still expect pkgB to be at `0.3.0` and pkgC to be at `3.3.4` at L1236-40. You'd need to copy/tweak the data for `pkgAdata` from L49 to use on L1174.

But, since you've already verified that crossing the "breaking change" boundary for `0.2.1` -> `0.3.0` for a dependency works then it's probably not necessary so I'll let you decide if you want to include it.